### PR TITLE
Change 4 lines to disable some recipes in recipes.cfg

### PR DESCRIPTION
**Pull request by Prometheus0000**
Disable iridium shapeless uncrafting https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/4860
Disable nether quartz shapeless uncrafting https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/7956
Disable nether quartz maceration (another recipe is imported from IC2, so you can still macerate) https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/7956
Disable clay block maceration (another recipe is imported from IC2, so you can still macerate) https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/8036

I did not remove the 9 dust to 1 block compression recipe so you can still get back to a block if you _really_ want to. Also, don't blame me for IC2's recipes losing half the mass when macerating clay blocks or balls.

Should prevent duping of clay and nether quartz. Also thanks to glease for help with the config file.

PS: I had to update the config file first, since there was a ton of stuff missing. There was also a bunch of stuff in my test instance not in the main pack for some reason, possibly added by obama or something, though they seemed like GT++ materials. So it probably should be updated again after obama is added and GT++ removed.

### CODE REVIEW

**Review by Dream-Master — CHANGES_REQUESTED** [review]
use 0 instead of minus values

File: config/GregTech/Recipes.cfg [hunk 1]
```diff
@@ -140857,8 +140857,8 @@ pulveriser {
     I:blockChromiumDioxide_756=756
     I:blockChromo-Alumino-Povondraite_400=400
     I:blockChromo-Alumino-Povondraite_882=882
-    I:blockClay_135=135
-    I:blockClay_400=400
+    I:blockClay_135=-135
```
Comment (Dream-Master) [inline config/GregTech/Recipes.cfg:140860]: I:blockClay_135=0

File: config/GregTech/Recipes.cfg [hunk 2]
```diff
@@ -140857,8 +140857,8 @@ pulveriser {
...
+    I:blockClay_400=-400
```
Comment (Dream-Master) [inline config/GregTech/Recipes.cfg:140861]: I:blockClay_400=0


and so on

**Review by Dream-Master — CHANGES_REQUESTED** [review]

File: config/GregTech/Recipes.cfg (hunk 1 above)
Comment (Prometheus0000) [reply config/GregTech/Recipes.cfg:140860]: > set it to any nonpositive value will disable the recipe

Is what glease said. 0 is positive, so I dunno if it works, but I'll try it

Comment (Dream-Master) [reply config/GregTech/Recipes.cfg:140860]: i did all recipes i dont want with 0 and it works

Comment (Prometheus0000) [reply config/GregTech/Recipes.cfg]: It works, so I changed them. Personally, I think - is better since it's possible to search for, but whatever.

**Review by Dream-Master — APPROVED** [review]
cant see the change becaue of 30k changes. Will see ingame

**Pull request merged**